exception_reporter.gemspec

```diff
@@ -22,7 +22,7 @@ Gem::Specification.new do |spec|
 
   spec.add_dependency 'telemetry_logger'
   spec.add_dependency 'dependency'
-  spec.add_dependency 'null_attr'
+  spec.add_dependency 'subst_attr'
```

Comment:
@litch This looks like a mistake or an oversight. @kizzle102 followed the existing pattern, but this can't possibly work.